src/integrationTest/java/org/opensearch/test/framework/TestIndex.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021-2022 floragunn GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.test.framework;
+
+import org.opensearch.action.admin.indices.create.CreateIndexRequest;
+import org.opensearch.client.Client;
+import org.opensearch.common.settings.Settings;
+
+public class TestIndex {
+
+    private final String name;
+    private final Settings settings;
+    
+    public TestIndex(String name, Settings settings) {
+        this.name = name;
+        this.settings = settings;
+        
+    }
+
+	public void create(Client client) {
```

Comment:
Looks like whitespace is not aligning well.  Could you update the spotless settings to do this automatically for this new source tree?

Docs - https://github.com/diffplug/spotless/tree/main/plugin-gradle

Reply:
We don't understand what you mean. Can you explain that?

Reply:
It looks like sometimes tabs are being used, other times spaces, and the settings are sometimes 8spaces vs 4spaces.  For this new code the style should be consistent between the files.

My recommendation is that you could alter the configuration of the spotless gradle plugin to handle aligning the code formatting at build time that will ensure consistancy.

Reply:
I added new spotless config for integrationTest - only tabs are allowed.